draft-ietf-masque-connect-ip.md

```diff
@@ -796,6 +789,8 @@ IP packets are encoded using HTTP Datagrams with the Context ID set to zero.
 When the Context ID is set to zero, the Payload field contains a full IP packet
 (from the IP Version field until the last byte of the IP Payload).
 
+## HTTP Datagram Handling {#payload-handling}
```

Comment:
```suggestion
# IP Packet Handling
```

Reply:
Yeah I was debating between the two. Done.